dora/src/semck/stdlib.rs

```diff
@@ -239,6 +263,31 @@ fn internal_struct(
     struct_id
 }
 
+fn internal_annotation(
+    vm: &mut VM,
+    namespace_id: NamespaceId,
+    name: &str,
+    internal_annotation: Modifier,
+) -> AnnotationId {
+    let name = vm.interner.intern(name);
+    let mut annotations = vm.annotations.lock();
+    let id: AnnotationId = annotations.len().into();
+    let annotation = Annotation {
```

Comment:
I am a bit confused about this code. In this file we usually only find e.g. classes or functions by name to get e.g. their IDs or set some specific properties. We generally create such data structures in src/semck/globaldef.rs.

Reply:
I think the visitor in globaldef needs to be extended at a later stage to handle annotations that aren't "special", but at the same time we need to deal with these internal annotations early on – do you have a suggestion how it would work better?

(We could extend the visitor now and call `parse_file` with explicit arguments (or something along the lines of `parse_bundled_stdlib` ... do you have preferences?))